doc/developer/testing.rst

```diff
@@ -9,6 +9,85 @@ Quick instructions
 To run the tests, simply execute the following command. Omit ``SR_PYTHON=On`` or ``SR_FORTRAN=On`` if you do not wish to test those languages:
    ``make test SR_PYTHON=On SR_FORTRAN=On``
 
+####################
+Running single tests
+####################
+
+With the increased automation in the SmartRedis test system, it is now more complex to run
+an individual test case. The steps to follow are as follows:
+
+1. Identify the test you wish to run.
+
+This is most easily done by running the full test suite
+and noting the name and path to the test. For example to run the ``test_config_set_command_DNE``
+from within ``test_nonkeyed_cmd.py``, we'll save off the path:
+``tests/python/test_nonkeyed_cmd.py::test_config_set_command_DNE``
+
+.. code-block:: bash
+
+  tests/python/test_nonkeyed_cmd.py::test_flushdb_command PASSED                                                 [ 87%]
+  tests/python/test_nonkeyed_cmd.py::test_config_set_get_command PASSED                                          [ 88%]
+  tests/python/test_nonkeyed_cmd.py::test_config_set_command_DNE PASSED                                          [ 88%]
+  tests/python/test_nonkeyed_cmd.py::test_config_get_command_DNE PASSED                                          [ 89%]
+  tests/python/test_nonkeyed_cmd.py::test_save_command PASSED                                                    [ 89%]
+
+2. Launch a backend database.
+
+Here, we'll set up a small Redis cluster with three nodes on the local
+host at ports `6390`, `6391`, and `6392`, using the ``launch_redis.py`` script located in
+the ``utils`` folder. If the ``nodes`` argument is not one, the launched nodes will be
+clustered together; otherwise a standalone backend will be created.
```

Comment:
- consider inverting the note about nodes to avoid starting with a negation:
- consider a note block

```
.. note:: 
  A standalone backed is created when the value of ``nodes`` is 1. Values greater than 1 result in a clustered set of database instances.
```

Reply:
Moved to a note, but kept the ordering -- clustering should be the default choice